R/hypothesize.R

```diff
@@ -0,0 +1,93 @@
+# hypothesize.R
+#' @title hypothesize
+#' @author Nada Elnour, \email{nada.elnour@@mail.utoronto.ca}
+#'
+#' \code{hypothesize} generates an annotated hypothesis graph whose nodes are
+#' components of \code{mySys} and edges are GGI interpretations in either
+#' \code{EMAP} or \code{GMAP}
+#'
+#' @param network A dataframe of physically-interacting system components with
+#' their genetic interactions and
+#' @param ppi_ggi An optional dataframe to specify subset of \code{mySys} for
+#' which both PPI and GGI data is available
+#'
+#' @importFrom dplyr inner_join
+#' @import visNetwork
+#' @examples
+#' mySys <- getSysInteractions("SLIGR", criterion = "stringent")
+#' mySys2 <- getSysInteractions("SLIGR", criterion = "relaxed")
+#' hypothesize(mySys)
+#' hypothesize(mySys2, mySys)
+#'
+#' @export
+hypothesize <- function(network, ppi_ggi = NULL) {
+    # make interpretation maps
+    EMAP <- makeEMAP()
+    GMAP <- makeGMAP()
+
+    # plot the network
+    visualizeInteractions(network, EMAP, ppi_ggi, GMAP)
+  }
+
+#' @title visualizeInteractions
+#' @author Nada Elnour, \email{nada.elnour@@mail.utoronto.ca}
+#'
+#' A helper function of \code{hypothesize} that plots the hypothesis graph
+#'
+#' @inheritParams hypothesize
+#' @param gmap The dataframe obtained from \code{makeGMAP}.
+#' @param emap The dataframe obtained from \code{makeEMAP}.
```

Comment:
The @return is missing.

Also, as above, changing the parameters format

Reply:
fixed.

Reply:
Great! after fixing every thing, please submit a new pull request (that way the tag for "changes requested" will be removed)